test/Sentry.OpenTelemetry.Tests/SentrySpanProcessorTests.cs

```diff
@@ -183,12 +182,56 @@ public void OnStart_WithParentSpanId_StartsChildSpan()
         }
     }
 
+    [Fact]
+    public void OnStart_WithSentryParentSpanId_StartsChildSpan()
```

Comment:
That should work the other way around as well, right?

Reply:
It does yes... I'll add another test.